web/rest/user/config/api/raw/provider.yml

```diff
@@ -111,12 +115,56 @@ Ivoz\Provider\Domain\Model\Voicemail\Voicemail:
         tags:
         - My
         parameters: []
+
+    get_my_voicemails:
+      method: 'GET'
+      path: '/my/voicemails'
+      route_name: 'get_my_voicemails'
+      operation_normalization_context: !php/const Ivoz\Provider\Domain\Model\Voicemail\VoicemailDto::CONTEXT_MY_VOICEMAILS
+      normalization_context:
+        groups: [
+          !php/const Ivoz\Provider\Domain\Model\Voicemail\VoicemailDto::CONTEXT_MY_VOICEMAILS
+        ]
+      swagger_context:
+        produces:
+        - 'application/json'
+        - 'application/ld+json'
+        tags:
+        - My
+        parameters: []
+  properties:
+    name:
+      attributes:
+        swagger_context:
+          readOnly: true
+    email:
+      attributes:
+        swagger_context:
+          readOnly: true
   attributes:
     access_control: '"ROLE_COMPANY_USER" in roles'
+    read_access_control:
+      ROLE_COMPANY_USER:
+        user:
+          eq: "user.getId()"
```

Comment:
This is ignored on custom endpoints. Just implement the validations in the controller / app service

Reply:
/my/voicemails only returns voicemails for the logged user, so it‘s already validated, but removing this would affect /provider/voicemails/{id} get and put operations.